lib/straight/order.rb

```diff
@@ -163,6 +163,7 @@ def status_changed?
         @status_changed
       end
 
+      # @deprecated
       def tid
```

Comment:
Anyway, this will be just moving code from one place to another, so let's keep it for a few releases.